src/solvers/flattening/boolbv.h

```diff
@@ -79,7 +79,7 @@ class boolbvt:public arrayst
     SUB::post_process();
   }
 
-  enum class unbounded_arrayt { U_NONE, U_ALL, U_AUTO };
+  enum class unbounded_arrayt { U_AUTO, U_ALL, U_AUTO };
```

Comment:
This does not seem to be part of this PR.  Please have a look at https://github.com/diffblue/cbmc/pull/6232 .

Reply:
Sorry @martin-cs I send it here by mistake.